src/middlewares/noAuth.middleware.js

```diff
@@ -0,0 +1,23 @@
+const { ApiError } = require('../exceptions/api.error');
+const jwtService = require('../services/jwt.service');
+
+const noAuthMiddleware = (req, res, next) => {
+  const authorization = req.headers['authorization'] || '';
+  const [, token] = authorization.split(' ');
+
+  if (authorization || token) {
+    throw ApiError.forbidden();
+  }
```

Comment:
The condition 'if (authorization || token)' will always be true if 'authorization' is not an empty string, even if 'token' is undefined. This logic will throw a forbidden error even when there is no token. Consider checking only for the presence of a token.